### PR TITLE
Add ResNetVD presets

### PR DESCRIPTION
**Pull request by gowthamkpr**
Adds the presets for the ResNet_vd variant, including the Kaggle handles. Parameter counts include the classification head.

### CODE REVIEW

File: keras_hub/src/models/resnet/resnet_presets.py [hunk 1]
```diff
@@ -79,4 +79,145 @@
...
+                "resolution with knowledge distillation and AutoAugment."
+            ),
+            "params": 25629512,
+            "official_name": "ResNet",
+            "path": "resnet",
+            "model_card": "https://arxiv.org/abs/1812.01187",
+        },
+        "kaggle_handle": "kaggle://kerashub/resnetvd/keras/resnet50_vd_ssld_v2",
+    },
+    "resnet_vd_50_ssld_v2_fix_imagenet": {
```
Comment (divyashreepathihalli) [inline keras_hub/src/models/resnet/resnet_presets.py:152]: what is `fix`?

Comment (gowthamkpr) [reply keras_hub/src/models/resnet/resnet_presets.py:152]: "`Fix_ResNet50_vd_ssld_v2` stopped all parameter updates of ResNet50_vd_ssld_v2 except the FC layer,and fine-tuned on ImageNet1k dataset" (from https://paddleclas.readthedocs.io/en/latest/models/ResNet_and_vd_en.html)

Comment (divyashreepathihalli) [reply keras_hub/src/models/resnet/resnet_presets.py:152]: thanks!

**Comment by divyashreepathihalli** [comment]
Please fix the GPU tests

**Comment by divyashreepathihalli** [comment]
Please fix the GPU tests

File: tools/checkpoint_conversion/convert_resnet_vd_checkpoints.py [hunk 2]
```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env python3
+
```
Comment (divyashreepathihalli) [inline tools/checkpoint_conversion/convert_resnet_vd_checkpoints.py:2]: can you add instructions to run this file on the top of the file?

Comment (gowthamkpr) [reply tools/checkpoint_conversion/convert_resnet_vd_checkpoints.py:2]: Done.

**Review by divyashreepathihalli — APPROVED** [review]

**Pull request merged**